### PR TITLE
Create ISSUE_TEMPLATE.md

### LINKED ISSUE #436: Create an issue Template for this repo
Hey,

Do we want an issue template for this repo ?

If so, what do we want inside ? ATM, I see:

* Node.js Version
* Scope (install, code, runtime)
* Package (and version) (if relevant)

Any ideas ?
cc @nodejs/evangelism

### PR DESCRIPTION
**Pull request by vdeturckheim**
Closes #436

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Please remove this line and fill the template
```
Comment (gibfahn) [inline ISSUE_TEMPLATE.md]: Should these be `<!--  -->` style comments (as in https://raw.githubusercontent.com/nodejs/node/master/.github/ISSUE_TEMPLATE.md)?

**Comment by gibfahn** [comment]
Also (based on [nodejs/node](https://raw.githubusercontent.com/nodejs/node/master/.github/ISSUE_TEMPLATE.md)) does this template need to be in a `.github` folder to get picked up?

**Comment by vdeturckheim** [comment]
@gibfahn you are right. Thanks for pointing that ! I'll do this ASAP

**Comment by vdeturckheim** [comment]
The `.github`is optional I think ( https://github.com/blog/2111-issue-and-pull-request-templates  )

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!-- Please remove this line and fill the template -->
+
+* **Node.js Version**:
+* **OS**:
+* **Scope (install, code, runtime, meta, other?)**:
+* **Package (and version) (if relevant)**:
```
Comment (gibfahn) [inline ISSUE_TEMPLATE.md]: Maybe `Package` -> `Module`?

Comment (vdeturckheim) [inline ISSUE_TEMPLATE.md]: Done ;)

**Comment by vdeturckheim** [comment]
@gibfahn can I merge?

**Comment by gibfahn** [comment]
It'd be good to get some reviews from people who've been in this repo for more than two days :), cc/ @nodejs/evangelism

**Comment by thefourtheye** [comment]
This looks good enough. LGTM

**Comment by bnb** [comment]
I don't see any problems - LGTM. 👍

**Comment by vdeturckheim** [comment]
Merged, thanks guys

**Pull request merged**